src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -188,18 +199,14 @@ html[data-theme="light"] {
 
 details.sd-dropdown {
   // Remove all borders to over-ride SD behavior, and we'll add our own later
-  border: 0px !important;
-  summary.sd-card-header {
-    border: 0 !important;
-    & + div.sd-summary-content {
-      border: 0;
-    }
```

Comment:
The linter complained about duplicate selectors and it felt safe to move these rules below